light-system-programs/programs/verifier_program_one/Cargo.toml

```diff
@@ -17,12 +17,12 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = { git = "https://github.com/coral-xyz/anchor", rev = "1c6f86e5f7793ce6adefe9cbfa11939647c509ce" }
-anchor-spl = { git = "https://github.com/coral-xyz/anchor", rev = "1c6f86e5f7793ce6adefe9cbfa11939647c509ce" }
+anchor-lang = "0.28.0"
```

Comment:
Same here. This change is already on main branch https://github.com/Lightprotocol/light-protocol/blob/main/light-system-programs/programs/verifier_program_one/Cargo.toml#L20

Please rebase. I'm not sure why Github isn't complaining about your branch being outdated.

Reply:
the fun fact here is that I rebased but didn't get any conflict in the process